lib/God/ForkMode.js

```diff
@@ -92,15 +93,21 @@ module.exports = function ForkMode(God) {
           env      : pm2_env,
           detached : true,
           cwd      : pm2_env.pm_cwd || process.cwd(),
-          stdio    : ['pipe', 'pipe', 'pipe', 'ipc'] //Same as fork() in node core
+          stdio    : ['pipe', 'pipe', 'pipe'] //Same as fork() in node core
         }
 
         if (typeof(pm2_env.windowsHide) === "boolean") {
           options.windowsHide = pm2_env.windowsHide;
         } else {
           options.windowsHide = true;
         }
-
+        if(interpreter === 'node'){
```

Comment:
we recently add the issue of the interpreter not being strictly `node`, so i believe something like `(interpreter.indexOf('node') > -1)` would be better.